Standards/scs-XXXX-v1-default-rules-for-security-groups.md

```diff
@@ -0,0 +1,116 @@
++--------------------------+-------------+-----------+-----------+------------+-----------+-----------------------+----------------------+--------------------------------+-------------------------------+
+| ID                       | IP Protocol | Ethertype | IP Range  | Port Range | Direction | Remote Security Group | Remote Address Group | Used in default Security Group | Used in custom Security Group |
++--------------------------+-------------+-----------+-----------+------------+-----------+-----------------------+----------------------+--------------------------------+-------------------------------+
+| 47b929fd-9b39-4f22-abc5- | None        | IPv6      | ::/0      |            | egress    | None                  | None                 | True                           | True                          |
+| 7d4f64d10909             |             |           |           |            |           |                       |                      |                                |                               |
+| 92a79600-5358-4fef-a390- | None        | IPv4      | 0.0.0.0/0 |            | egress    | None                  | None                 | True                           | True                          |
+| 822665f46070             |             |           |           |            |           |                       |                      |                                |                               |
+| 93e35d0c-2482-4ec1-9fbd- | None        | IPv4      | 0.0.0.0/0 |            | ingress   | PARENT                | None                 | True                           | False                         |
+| fd8c9a21a04e             |             |           |           |            |           |                       |                      |                                |                               |
+| ed5cd662-add2-4e42-b0a7- | None        | IPv6      | ::/0      |            | ingress   | PARENT                | None                 | True                           | False                         |
+| 3b585d348820             |             |           |           |            |           |                       |                      |                                |                               |
++--------------------------+-------------+-----------+-----------+------------+-----------+-----------------------+----------------------+--------------------------------+-------------------------------+
+```
+
+## Related Documents
+
+The spec for introducing configurability for the default Security Groups Rules can be found [here](https://specs.openstack.org/openstack/neutron-specs/specs/2023.2/configurable-default-sg-rules.html).
+
+More about Security Groups as a resource in OpenStack can be found [here](https://docs.openstack.org/nova/latest/user/security-groups.html).
+
+## Conformance Tests
+
+The conformance tests should check for the absence of any ingress traffic rules in the `openstack security group rule list`.
```

Comment:
The conformance test should also create a new security group and check that, too.

Reply:
What would be the improvement of this? The default security group rules are the ones used to create the rules in security groups. Testing both would not test the standard but would test for bugs in the openstack code applying the default rules to the security groups.
PLUS: to test this through creating security groups, it would also be necessary to create a new project, to have a brand new default security group. We should be able to test everything as non-admin users.

Reply:
Because in the text as written, there is nothing which tests the configuration of the default security group rules, is there?

I might've missed some implicit test there, because I'm not deeply in the matter. But we are defining rules for the default security group, so that those rules are in effect should be tested.

Reply:
We discussed this: there was a mistake in the standard. I am testing the "openstack default security group rule list" endpoint, not the `openstack security group rule list`. I changed this.